src/parsing.js

```diff
@@ -14,14 +14,78 @@
 
 (function(internal, testing) {
 
-  var _numberValueRegex = /^\s*[-+]?(\d*\.)?\d+(e[-+]?\d+)?\s*$/;
+  // Extra backslashes because otherwise JS interprets them incorrectly.
+  var _numberValueRegexStr = '[-+]?(\\d*\\.)?\\d+(e[-+]?\\d+)?';
```

Comment:
Is there a reason this has an underscore in the name? Your knowledge of the style guide leads me to think yes :P